ethcore/src/executive.rs

```diff
@@ -34,6 +34,8 @@ pub use types::executed::{Executed, ExecutionResult};
 /// Maybe something like here: `https://github.com/ethereum/libethereum/blob/4db169b8504f2b87f7d5a481819cfb959fc65f6c/libethereum/ExtVM.cpp`
 const STACK_SIZE_PER_DEPTH: usize = 24*1024;
 
+const WASM_MAGIC_NUMBER: [u8; 4] = [0x00, 0x61, 0x73, 0x6d];
```

Comment:
This is just personal taste but if a byte array is supposed to be ASCII I'd rather write this as `*b"\0asm"` instead. It doesn't matter, but it might be something to keep in mind going forward.